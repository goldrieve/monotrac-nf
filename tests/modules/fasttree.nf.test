nextflow_process {

    name "Test Process FASTTREE"
    script "main.nf"
    process "FASTTREE"

    test("Should run tree generation") {

        when {
            process {
                """
                input[0] = file("${projectDir}/tests/data/aligned.fas")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }
    }
}