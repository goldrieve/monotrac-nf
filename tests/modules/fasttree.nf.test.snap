{
    "Should run tree generation": {
        "content": [
            {
                "0": [
                    "tree.nwk:md5,30c06057ba09228e29b8285953367a92"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.7"
        },
        "timestamp": "2025-10-06T12:21:21.202114469"
    }
}